### ISSUE TITLE
MOTD for a specific course

### ISSUE THREAD

**Issue by a18eliek** [HGustavs/LenaSYS#7965]
A message of the day for a specific course could be useful, this could highlight some important information to the students that they need to be aware about for that specific course.

One thing that also could be added to this could be a change in the background color, to highlight the importance of the message. 

This is how I would imagine this would look like:
![image](https://user-images.githubusercontent.com/49141837/78768840-cecac900-798c-11ea-900c-9193860cb15a.png)

This message should only show on the specific course, not on all of them.

**Comment by HGustavs** [comment]
Not a bad idea

**Comment by HGustavs** [comment]
It is a bit too unobtrusive.
Unclear to me if that is good or bad. 

Do we want to shout to force the user to click or do we want something softer.

**Comment by a18jonkl** [comment]
@HGustavs I have looked into this problem and a solution is to add column names "motd" to table "vers". Is it okey to add columns to the DB?

Is there another way to store specific messages for courses?

**Comment by HGustavs** [comment]
yes it is possible to add columns.

We do not want the existing stuff to break but it seems like we can make this change incremental.

**Comment by HGustavs** [comment]
if this column is not present if it still works it is more than fine, if the column is present but with old data, and it also works in that case... I dont think we need to test any other cases?

**Comment by a18jonkl** [comment]
There is now a solution that works with and without the column ‘motd’ in table ‘vers’. If the column is not presented than nothing will be stored and there is no old data to find. There is a MOTD text field in ‘Edit Course Version’ that cannot be left empty even without the ‘motd’ column. The text you enter will not be saved anywhere and it will also give you an error that the column is missing(This is just a warning to update the DB).

To make this possible there is some redundant code in `sectionedservice.php`. It checks the queries with ‘motd’ as a parameter and if it fails it will run the original query without the ‘motd’. The best solution to this is to fix so everyone has this in their tables and then remove the redundant code.

The MOTD for each course will be displayed until the person logout or if the MOTD is changed. This is handled by cookies. A cookie called ‘’motd” will activate when the user presses the close button for the MOTD dialog and store the variable `vers` and `versname` from table 'vers' to make it course specific. This is stored as a string separated by commas.

I have tested everything with and without the column ‘motd’ and everything seems to be working. This will need some extra testing, so everything is checked. I have also tested changing the `init_db.sql` with the column `motd` and it seemed to work fine(This is not committed into the weekly branch). 

Is there a case that I have not explored in my solution? @HGustavs
